### PR TITLE
read only workflows

### PR DESCRIPTION
**Pull request by peicodes**
https://vellum-preview-d8733154-af50-473f-a048-718d21628e4a.docs.buildwithfern.com/changelog/2024/2024-03#read-only-workflow-diagrams

### CODE REVIEW

File: fern/docs/content/change-log/2024-03.mdx [hunk 1]
```diff
@@ -2,6 +2,14 @@
 title: Changelog | March, 2024
 ---
 
+## Read-only Workflow Diagrams
+
+_March 22th, 2024_
+
+You can now see a read-only view of workflow diagrams for workflow deployment executions, workflow test case executions, and workflow releases. You can access the diagram by clicking the "Graph View" icon tab on the top right.
```
Comment (noanflaherty) [inline fern/docs/content/change-log/2024-03.mdx:9]:
```suggestion
You can now see a read-only view of workflow diagrams for Workflow Deployment Executions, Workflow Test Case Executions, and Workflow Releases. You can access the diagram by clicking the "Graph View" icon tab on the top right.

This is particularly helpful if you want to visualize what your Workflow looked like at that time, as well as visualize the execution path your Workflow took.
```

File: fern/docs/content/change-log/2024-03.mdx [hunk 2]
```diff
@@ -2,6 +2,14 @@
...
+
+![Workflow Deployment Execution](https://storage.googleapis.com/vellum-public/help-docs/workflow-deployment-execution.png)
```
Comment (noanflaherty) [inline fern/docs/content/change-log/2024-03.mdx:11]: Haha let's avoid a screenshot that talks about the red hysteria...

Also would be nice if:
1) The entire workflow graph was in view in the canvas.
2) There was a slightly more interesting workflow. 

Here's a screenshot we can use instead:

**Review by noanflaherty — APPROVED** [review]
Some quick feedback, then let's 🚢

File: fern/docs/content/change-log/2024-03.mdx (hunk 2 above)
Comment (peicodes) [reply fern/docs/content/change-log/2024-03.mdx:11]: > Haha let's avoid a screenshot that talks about the red hysteria...

LOL sorry I totally forgot that was my execution

**Pull request merged**